commands/command_push.go

```diff
@@ -45,7 +46,7 @@ func uploadsBetweenRefAndRemote(remote string, refs []string) *lfs.TransferQueue
 		if len(refs) == 0 {
 			pointers := scanAll()
 			Print("Pushing objects...")
-			return uploadPointers(pointers)
+			return uploadPointers(pointers, strings.Join(refs, ","))
```

Comment:
True, on the server side we are not interested in the name of local branch, we just want to know where you want to push (put) your changes.

Will remove my modifications from the command_push.go